SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -157,6 +158,7 @@
   @JsonProperty
   private String dockerPrefix = "se-";
 
+  @NotNull
```

Comment:
There's a bunch of these below -- can you replace all `@NotNull`s with `@Min`s that make sense?